code/game/gamemodes/endgame/scoreboard/scoreboard.dm

```diff
@@ -0,0 +1,317 @@
+var/global/datum/controller/gameticker/scoreboard/score = new()
+
+///datum/controller/gameticker/scoreboard/New()
```

Comment:
Is there a reason this is commented our rather than deleted?

Reply:
no